distribution/osbuild-composer-cloud.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=OSBuild Composer cloud
+After=multi-user.target
+Requires=osbuild-composer-cloud.socket
+
+[Service]
+Type=simple
+ExecStart=/usr/libexec/osbuild-composer/osbuild-composer-cloud
+CacheDirectory=osbuild-composer-cloud
+StateDirectory=osbuild-composer-cloud
+WorkingDirectory=/usr/libexec/osbuild-composer/
+User=_osbuild-composer
```

Comment:
`_osbuild-composer` user is created by `osbuild-composer` package but `osbuild-composer-cloud` doesn't depend on it. I tried this:
```
dnf install osbuild-composer-cloud
systemctl start osbuild-composer-cloud.service
```

and systemd was unable to start the service because `_osbuild-composer` user doesn't exist as the `osbuild-composer` package isn't installed.